po/zh_CN.po

```diff
@@ -0,0 +1,1948 @@
+# Simplified Chinese translations for com.github.johnfactotum.Foliate package.
+# Copyright (C) 2019 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.johnfactotum.Foliate package.
+# Blanc Chan cdh4227@qq.com, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.johnfactotum.Foliate\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-21 15:05+0800\n"
+"PO-Revision-Date: 2020-05-12 13:55+0800\n"
+"Last-Translator: Blan Chan cdh4227@qq.com\n"
+"Language-Team: none\n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:3 src/main.js:105
+#: data/com.github.johnfactotum.Foliate.appdata.xml.in:6
+msgid "Foliate"
+msgstr ""
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:4
+msgid "eBook Viewer"
+msgstr "电子书查看器"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:5
+msgid "View eBooks"
+msgstr "阅读电子书"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:9
+msgid "com.github.johnfactotum.Foliate"
+msgstr ""
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:12
+msgid "Ebook;Book;EPUB;Viewer;Reader;"
+msgstr "电子书;图书;EPUB;查看器;阅读器"
+
+#: src/main.js:80
+msgid "All Files"
+msgstr "所有文件"
+
+#: src/main.js:84
+msgid "E-book Files"
+msgstr "电子书文件"
+
+#: src/main.js:90
+msgid "Open File"
+msgstr "打开文件"
+
+#: src/main.js:104#, fuzzy
+msgid "translator-credits"
+msgstr "翻译者致谢"
+
+#: src/main.js:106 data/com.github.johnfactotum.Foliate.appdata.xml.in:7
+msgid "A simple and modern eBook viewer"
+msgstr "简洁现代的电子书查看器"
+
+#: src/window.js:122
+msgid "Leave fullscreen"
+msgstr "退出全屏"
+
+#: src/window.js:125 src/ui/mainMenu.ui:91
+#: data/com.github.johnfactotum.Foliate.gschema.xml:200
+msgid "Fullscreen"
+msgstr "全屏"
+
+#: src/window.js:183 src/window.js:184
+#, javascript-format
+msgid "of %d"
+msgstr "共 %d"
+
+#: src/export.js:31
+#, javascript-format
+msgid "<p>Annotations for</p><h1>%s</h1><h2>By %s</h2>"
+msgstr "<h1>《%s》</h1><p>的注释</p><h2>（%s 注）</h2>"
```

Comment:
This doesn't seem right. The second `%s` contains the author of the book, not the author of the annotations. Admittedly, the English text itself is ambiguous and confusing. But anyway that's just how it is now.